ocpmodels/trainers/forces_trainer.py

```diff
@@ -344,7 +348,7 @@ def predict(
         )
         return predictions
 
-    def train(self):
+    def train(self):  # noqa: C901
```

Comment:
Do we need this?

Reply:
`forces_trainer.py` doesn't pass the flake commit hook because it's "too complex", this is a way around it. Maybe we can add this in the `.pre-commit` file instead to exclude this pattern.

Reply:
I can double check but flake8 was failing because the train() function is too complex (too many if statements). @mshuaibii did you see that when you added an if statement to the train() function in #226? Or has this been resolved another way?

Reply:
Yeah I got similar errors with flake8 when I modified `forces_trainer`, I just overrode it at commit time (it's not tested at the CI-level)

Reply:
ok, I will remove and override for now and we can look into adding to `.pre-commit`